### DISCUSSION TITLE
Help regarding triggering completion

### DISCUSSION THREAD

**Discussion by uldrim** [neoclide/coc.nvim#4010] [Q&A] [answered]
Hi,
I am facing an issue while triggering completion. (Ubuntu 22.04)
I have remapped trigger completion to <Tab> using this code provided here https://github.com/neoclide/coc.nvim/wiki/Completion-with-sources#use-tab-or-custom-key-for-trigger-completion.

What I am facing is that the tab is switching to the second suggestion in the suggestion list, skipping the first, the first is highlighted initially.
I am unable to trigger the completion for the first suggestion, i have to press a lot of tabs until it starts from second suggestion to the last suggestion and then comes back to the first.
I would like to use the first suggestion somehow when i press tab.
Please help me in doing that.

**Comment by chemzqm** [comment, marked as answer]
You have to confirm completion or enable suggest.noselect
